frontend/server/src/DAO/Contests.php

```diff
@@ -634,6 +634,113 @@ final public static function getContestsParticipating(
         ];
     }
 
+    /**
+     * Returns the next contest a user registered for.
+     *
+     * @return ContestListItem
```

Comment:
Acá también podríamos estar regresando `null`:
```suggestion
     * @return ContestListItem|null
```